core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -122,6 +122,8 @@ var (
 type EthConfirmer struct {
 	utils.StartStopOnce
 
+	o ORM
+
```

Comment:
nit: can we keep a standard parameter name? it looks like you used `orm` in the resender and `o` here

Reply:
will do, i was going between o and orm but I think orm is more descipritve